### ISSUE TITLE
[suggestion] use icons for fileformat

### ISSUE THREAD

**Issue by coxackie** [hoob3rt/lualine.nvim#51]
Instead of the boring `unix`, `dos` and `mac` for fileformat, one may use devicons (since they are also used for file types). More precisely, from `https://www.nerdfonts.com/cheat-sheet`, check out `nf-dev-linux`, `nf-dev-windows` and `nf-dev-apple`. Just eye-candy, but, hey, why not? (I do not think both icon and text should be provided. Icons if the possibility is there, text for fallback.)

**Comment by shadmansaleh** [comment]
Seems like a nice idea

**Comment by coxackie** [comment]
I think it will make it look prettier, and also will relieve some space. I am debating on whether `nf-pl-current_line` and `nf-ple-current_column` would be good to use in `location`. They do look good, but maybe they will add up on space...

**Comment by hoob3rt** [comment]
This is related to #16 and #45. Hopefully in the near future you will be able to set
```lua
lualine.icons = true
lualine.powerline_fonts = true
lualine.sections.lualie_a = { 'filetype' icons false } -- something along those lines
```

**Comment by hoob3rt** [comment]
resolved with #45
